store/sql_post_store.go

```diff
@@ -467,3 +467,27 @@ func (s SqlPostStore) Search(teamId string, userId string, terms string, isHasht
 
 	return storeChannel
 }
+
+func (s SqlPostStore) GetAll(channelId string) StoreChannel {
```

Comment:
seems like a dangerous method that people may call by mistake.  We should remove this and just have a paging call.  The paging call will be used other places like analytics jobs etc.